sdk/core/src/lib.rs

```diff
@@ -66,12 +66,15 @@
 ///     }
 /// }
 /// ```
-
```

Comment:
Can removal of unnecessary newlines be enforced/fixed using a linter rule?

Reply:
It probably is, unfortunately, we have a problem with our current pre-commit hooks and I have to disable them when splitting up large commits. The way they work right now, you have to add all current pending files to the current commit, so you can't break large commits apart. You have to plan it all in advance which is pretty challenging to anticipate. Maybe you can help me figure that out.

Reply:
Ah, that sounds like an ordeal. Okay, let's take note of that and come back to it. I think @jonshattuck was also having some trouble with the pre-commit hooks messing with files 🤔

Reply:
I added a task card in Coda to track this work 👍